src/content/developers/docs/layer-2-scaling/index.md

```diff
@@ -69,17 +69,17 @@ The sidechain where ZK rollups happen can be optimised to reduce transaction siz
 
-| Pros                                                                                                               | Cons                                                                                                                                                                                                              |
-| ------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| No delay as proofs are already considered valid when submitted to the main chain.                                  | Limited to simple transfers, not compatible with the EVM.                                                                                                                                                         |
-| Less vulnerable to the economic attacks that [Optimistic rollups](#optimistic-pros-and-cons) can be vulnerable to. | Validity proofs are intense to compute – not worth it for applications with little on-chain activity.                                                                                                             |
-|                                                                                                                    | Slower subjective [finality](/glossary/#finality) time (10-30 min to generate a ZK proof) (but faster to full finality because there is no dispute time delay like in [Optimistic rollups](#optimistic-rollups)). |
+| Pros                                                                                                              | Cons                                                                                                  |
+| ----------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
+| Faster finality time since the state is instantly verified once the proofs are sent to the main chain.            | Some don't have EVM support.                                                                          |
+| Not vulnerable to the economic attacks that [Optimistic rollups](#optimistic-pros-and-cons) can be vulnerable to. | Validity proofs are intense to compute – not worth it for applications with little on-chain activity. |
+| Secure and decentralized, since the data that is needed to recover the state is stored on the layer 1 chain.      |                                                                                                       |
 
 #### Use ZK rollups {#use-zk-rollups}
 
 - [Loopring](https://loopring.org/#/)
 - [Starkware](https://starkware.co/)
-- [Matter Labs zkSync](https://matter-labs.io/)
+- [Matter Labs zkSync](https://zksync.io/)
```

Comment:
Is there a reason for this link switch when the former still works?

Reply:
Matter Labs is the company.
zkSync is the zkRollup solution itself.

I believe that most users will be more interested in zkSync than the company behind it.

Reply:
Ok cool, I'd have to agree.